.vscode/settings.json

```diff
@@ -25,5 +25,10 @@
   },
   "[yaml]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
+  "[javascript][typescript]": {
+    "editor.codeActionsOnSave": {
+      "source.fixAll": "explicit"
+    }
```

Comment:
For VS code, especially with PnP, I think it's pretty standard? It's only project-specific settings. I actually was just thinking that the codeActionsOnSave settings should probably be removed from here, though; those are really a user preference and not particular to this project

Reply:
Is there an easy way for Code users to have their own settings that overlay these?

Reply:
Interestingly, the TypeScript project has template files with a comment suggesting that you maybe use them: https://github.com/microsoft/TypeScript/tree/main/.vscode

Reply:
Huh, that is interesting. The VS Code docs suggest that sharing these among developers is one of the intended use cases: https://code.visualstudio.com/docs/getstarted/settings#_workspace-settings, which is how I'd always understood it (it's why they specifically live in the project directory, rather than elsewhere). The settings here take precedence over user settings, but anything unset here will fall back to user settings. So I think we should probably remove things that are preferences, but not really required for the project to "work" (like the auto-fix-on-save), but leave things like the default formatter config and PnP configurations for typescript, eslint, and prettier, since those are unintuitive to set up and required to have any sort of working editor support

Reply:
That sounds fine to me.

Reply:
Done!